packages/akashify/.env.example

```diff
@@ -0,0 +1,16 @@
+# https://nft.storage/docs/#get-an-api-token
+NFT_STORAGE_API_KEY=
+
+# https://filebase.com
+BACKUP_BUCKET=
+BACKUP_KEY=
+BACKUP_SECRET=
+BACKUP_ENCRYPTION_PASSPHRASE=super_secret_passphrase
+BACKUP_HOST=https://s3.filebase.com
+# every 15 minutes
+BACKUP_SCHEDULE=*/15 * * * *
```

Comment:
I think we probably only need to backup nightly (say 1 AM).

Because it doesnt take very long to catch up from last night, so we dont need to to try and do it multiple times per hour.

Reply:
NM, I just got through reading: https://github.com/ovrclk/akash-postgres-restore/

So, when this `cron` container first spins up it will restore the db from the latest backup and then it will take a recurring backup. 

Since we dont have persistent storage we need frequent backups because we are using them to not lose data on pod restarts right now instead of emergency situations.

Reply:
We are still going to have data loss here and this seems brittle, so we should def come up with a different solution once we get persistent storage.